### PR TITLE
external-scheduling-proxy: fix markdown

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Changes proposed in this pull request:

- Fix markdown for external-scheduling-proxy

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sandrobonazzola 

This pull request needs review by: @MarSik

### CODE REVIEW

File: source/develop/release-management/features/sla/external-scheduling-proxy.md [hunk 1]
```diff
@@ -51,31 +50,32 @@ This feature will also allow new feaures to build on it.
...
-`       `</Filters>
-`       `<CostFunctions>
-`         `<CostFunction name="Memory"/>
-`       `</CostFunctions>
-`       `<Balances>
-`         `<Balance name="Memory"/>
-`       `<Balances>
-`     `</PolicyUnits>
-         
+{% highlight xml %}
```
Comment (MarSik) [inline source/develop/release-management/features/sla/external-scheduling-proxy.md:59]: This does not look right in the GitHub rendered form, not sure what the markdown processor you use will do.

**Comment by sandrobonazzola** [comment]
Here's the page rendered:

Screenshot_2020-10-09 oVirt External Scheduling Proxy

File: source/develop/release-management/features/sla/external-scheduling-proxy.md (hunk 1 above)
Comment (sandrobonazzola) [reply source/develop/release-management/features/sla/external-scheduling-proxy.md:59]: Attached screenshot

**Review by MarSik — APPROVED** [review]

**Pull request merged**